app/controllers/tournaments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class TournamentsController < ApplicationController
+
+  skip_before_action :verify_authenticity_token
+
+  def index
+  end
+
+  def create
+    Tournament.create!(company_name: params[:company_name], start_date: params[:start_date], end_date: params[:end_date])
+  end
+
+  def show
+    tournament = Tournament.find(params[:id])
```

Comment:
```suggestion
    Tournament.find(params[:id])
```
There's no need for tournament since you're not doing anything with it